test/src/e2e_vm_tests/test_programs/should_pass/stdlib/u256_div_test/src/main.sw

```diff
@@ -14,13 +14,14 @@ fn main() -> bool {
     assert(div_max_two.d == u64::max() >> 1);
 
     // Product of u64::MAX and u64::MAX.
-    let mut dividend = U256::from((0, 0, u64::max(), 1));
-    let mut res = dividend / max_u64;
+    let dividend = U256::from((0, 0, u64::max(), 1));
+    let res = dividend / max_u64;
     assert(res == U256::from((0, 0, 1, 0)));
 
-    dividend = U256::from((u64::max(), 0, 0, 0));
-    let mut res = dividend / max_u64;
-    assert(res == U256::from((1, 0, 0, 0)));
+    //TODO: @hal3e figure out why this is failing now
```

Comment:
This part of the test does not work. I need some help to debug it.

Reply:
I uncommented this and the test is working.

Reply:
Done.